src/utils/deprecations.ts

```diff
@@ -17,3 +17,5 @@ export const noModelDropSchema = deprecate(noop, 'Do not use Model.dropSchema. U
 export const movedSequelizeParam = deprecate(noop, 'The "sequelize" instance has been moved from the second parameter bag to the first parameter bag in "beforeAssociate" and "afterAssociate" hooks', 'SEQUELIZE0012');
 export const hooksReworked = deprecate(noop, 'Sequelize Hooks methods, such as addHook, runHooks, beforeFind, and afterSync… are deprecated in favor of using the methods available through "sequelize.hooks", "Sequelize.hooks" and "YourModel.hooks".', 'SEQUELIZE0013');
 export const doNotUseRealDataType = deprecate(noop, 'Sequelize 7 has normalized its FLOAT & DOUBLE data types, and made REAL redundant. FLOAT is now always an IEEE-754 single precision floating point, and DOUBLE a double-precision one. Use either instead of REAL.', 'SEQUELIZE0014');
+export const noSchemaParameter = deprecate(noop, 'The schema parameter has been deprecated, use a TableName object to specify the schema or set the schema globally in the options.', 'SEQUELIZE0015');
```

Comment:
I think these two should mention the method(s) where this was deprecated

Reply:
By just listing the method(s) or passing the method name to this and mention that in the message?

Reply:
You can't pass anything to this, and only the first deprecation notice gets printed by node (hence the SEQUELIZE0015 identifier), so by listing the affected method(s) here :)

Reply:
Updated in https://github.com/sequelize/sequelize/pull/15299/commits/4c93ebfc194c9058d5e0b5f172aa1d5b8374e22d